src/components/PlaylistModal/PlaylistModal.js

```diff
@@ -0,0 +1,91 @@
+import "./playlistModal.css";
+import CancelIcon from "@mui/icons-material/Cancel";
+import PlaylistAddIcon from "@mui/icons-material/PlaylistAdd";
+import { useState } from "react";
+import {
+  addToPlayList,
+  addVideoToPlayList,
+  removeVideoFromPlayList,
+} from "../../services";
+import { useData } from "../../context";
+import { isVideoInList } from "../../utils/";
+
+export const PlaylistModal = ({ video, showModal, setShowModal }) => {
+  const [playlistInput, setPlaylistInput] = useState({ title: "" });
+  const {
+    dataState: { createdPlaylist, playLists },
+    dataDispatch,
+  } = useData();
+
+  const addPlayLists = () => {
+    if (playlistInput.title === "") {
+      alert("Playlist name shouldn't be empty");
+    } else {

```

Comment:
avoid to user alert  you can use library to show this alert react-hot-toast 
or react-toastify

Reply:
yes, will use Toast in the next PR.